.gitignore

```diff
@@ -1,3 +1,2 @@
 /vendor
-composer.lock
-composer.phar
\ No newline at end of file
+composer.phar
```

Comment:
According to [the doc](https://getcomposer.org/doc/02-libraries.md#lock-file) the lock file is always ignored when the package us used at a dependency. However, it's always a good practise for better contributing :)